plugins/main/public/utils/applications.ts

```diff
@@ -618,7 +652,7 @@ const wazuhPluginSettings = {
   }),
   euiIconType: 'indexRollupApp',
   showInOverviewApp: false,
-  redirectTo: () => '/settings?tab=configuration',
```

Comment:
This should be in place for the redirect to work

Reply:
Done https://github.com/wazuh/wazuh-dashboard-plugins/pull/5997/commits/1fa1cd661038981db715e06ad0a9597ce1e13bda